src/meta/src/controller/catalog.rs

```diff
@@ -274,8 +278,30 @@ impl CatalogController {
             .into_tuple()
             .all(&txn)
             .await?;
+
+        let parallel_unit_to_worker = Self::get_parallel_unit_to_worker_map(&txn).await?;
+
         let fragment_mappings = get_fragment_mappings_by_jobs(&txn, streaming_jobs.clone()).await?;
 
+        let fragment_mappings = fragment_mappings
```

Comment:
Can we provide a helper function to wrap it, since this part will be used as long as the fragment mappings is transformed?